deploy/manifests/dev/us-east-2/tenant/storetheindex/instances/ago/deployment.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
```

Comment:
You will need to patch volume mounts here to add a separate one for adstore.

Reply:
Added a separate pvc definition and included it into here: https://github.com/ipni/storetheindex/pull/1177/files#diff-2dbb743179dee64e468ef645236b7a682b69daf647adf28ba24428b1356f31c2R29.